apps/src/codebridge/Console/Console.tsx

```diff
@@ -162,22 +162,24 @@ const Console: React.FunctionComponent = () => {
     }
   }, [fontSizeKey]);
 
-  // User preference for selected font size persists within a session
+  // User preference for selected console font size is saved on the backend
   // per signed-in user per app type (currently in pythonlab).
-  // TODO: update so that selected console font size will persist across sessions.
-  // Note that When the user selects a different font size from settings, fontSizeKey
-  // is updated alongside sessionStorage for sessionStorageKey.
+  // Note that when the user selects a different font size from settings, fontSizeKey
+  // is updated and saved on the backend.
   useEffect(() => {
-    const sessionStorageKey = `${appName}ConsoleFontSizeKey`;
-    const sessionStorage = tryGetSessionStorage(sessionStorageKey, false);
-    if (
-      sessionStorage &&
-      sessionStorage !== fontSizeKey &&
-      signInState === SignInState.SignedIn
-    ) {
-      dispatch(setConsoleFontSize(sessionStorage));
-    }
-  }, [signInState, fontSizeKey, appName, dispatch]);
+    const fetchFontSize = async () => {
+      if (signInState !== SignInState.SignedIn) {
+        return;
+      }
+      const savedConsoleFontSize =
```

Comment:
Would it make sense to move this plus the dispatch into a thunk?

Reply:
Oh yeah - nice idea! Updated.